tests/crypto_policy_test.go

```diff
@@ -183,10 +183,10 @@ func tryToAccessEndpoint(pod core.Pod, subpath string, port uint16, tlsConfig cl
 		subpath = "/" + subpath
 	}
 
-	// ssp-webhook-service.kubevirt.svc is used to access the endpoints we are testing.
+	// ssp-webhook-service.${deploymentNamespace}.svc is used to access the endpoints we are testing.
 	// It is used here for the metrics as well for simplicity, as it is the CN in the ca_cert
 	// and the metrics just sit on a different port on the same pod.
-	attemptedUrl = fmt.Sprintf("https://ssp-webhook-service.kubevirt.svc:%d%s", port, subpath)
+	attemptedUrl = fmt.Sprintf("https://ssp-webhook-service.%s.svc:%d%s", strategy.GetSSPDeploymentNameSpace(), port, subpath)
```

Comment:
I am unsure about the expected URL in OpenShift, is it
ssp-operator-service.openshift-cnv.svc
or
ssp-webhook-service.openshift-cnv.svc
?

Reply:
`ssp-webhook-service` is the only `Service` created by the manifests found in `config/webhook`, so `ssp-webhook-service.${deploymentNamespace}.svc` should be right.

Reply:
I think when the webhook is created by OLM, the service may have a different name. But I haven't tried it.

Reply:
Yes, this is also an issue. I will introduce another getter similar to the NS shortly.